proxy/src/control/pb.rs

```diff
@@ -232,258 +175,3 @@ impl<'a> TryFrom<&'a Event> for common::TapEvent {
         Ok(tap_ev)
     }
 }
-
-impl<'a> TryFrom<&'a common::http_method::Type> for http::Method {
```

Comment:
Why is it suddenly OK to delete all of this? I'm not saying it isn't OK, but it's not clear what this refactoring has done to make it OK to delete all this. Is there any negative effect of deleting all this?

Reply:
it's been moved with the grpc bindings